lips/lip-1.mdx

```diff
@@ -493,27 +425,188 @@ Represents a national ID.
+
+<img alt="Command Exchange" src={useBaseUrl('img/LIP1-PaymentBusinessFlow.png')} />
+
+## Steps of the protocol: The KYC Exchange Flow
+
+The basic KYC exchange flow starts with the Sender including KYC information and requesting KYC information by the receiver. In the straight-forward case the Receiver is satisfied with this information and responds with a command providing the receiver KYC information, and recipient signature. The Sender can then finalize the payment and also settle it on-chain (status sequence SINIT -> RSEND -> READY). At each step Sender or Receiver may also abort the payment.
+
+### Start -> SINIT (Step 1)
+The Sender issues `PaymentCommand` to create initial payment.
+
+The sender creates a payment object, that they believe requires KYC information exchange. The payment command includes a full payment object including all mandatory fields and the following optional fields populated as:
+
+* `Sender.Status.Status` = `need_kyc_data`.
+* `Sender.kyc_data` = The `KycDataObject` representing the sender.
+* `Receiver.Status.Status` = `none`
+
+### SINIT -> RSEND (Step 4)
+The Receiver issues `PaymentCommand` to update an existing payment.
+
+The receiver VASP examines the `Sender.kyc_data` object, and is satisfied that given the sender information the payment can proceed. It responds with a Payment Command that includes:
+
+* `Receiver.Status.Status` = `ready_for_settlement`
+* `Receiver.kyc_data` = The `KycDataObject` representing the receiver.
+* `recipient_signature` = a valid recipient signature.
+
+### SINIT -> RABORT (Step 9)
+The Receiver issues `PaymentCommand` to update an existing payment.
+
+The receiver VASP examines the sender KYC information and is either not satisfied the payment can proceed, needs more time to process the KYC information, or requires additional information to determine if the payment can proceed. It includes a command to abort the payment with an appropriate error code.
+
+* `Receiver.Status.Status`: `abort`
+* `Receiver.Status.abort_code`: one of `no-kyc-needed` or `rejected`.
+
+The sender can initiate a payment on-chain in case of an abort with `no-kyc-needed`.
+
+### RSEND -> READY (Step 7)
+
+The Sender issues `PaymentCommand` to update an existing payment.
+
+The Sender VASP examines the KYC information from the Receiver and is satisfied the payment can proceed.
+
+* `Sender.Status.Status`: `ready_for_settlement`
+
+The payment can be executed on-chain by the sender (or settled in any other way).
+
+### RSEND -> SABORT (Step 9)
+
+The Sender issues `PaymentCommand` to update an existing payment.
+
+The sender VASP requires more time, has a soft match for the receiver or cannot proceed with the payment. It issues an abort command:
+
+* `Sender.Status.Status`: `abort`
+* `Sender.Status.abort_code`: one of `pending`, `soft-match`, or `rejected`.
```

Comment:
why "soft-match" is abort code? what is difference comparing with sender send status = soft-match?